docs/resources/aws_ssm_resource_compliance_summaries.md

```diff
@@ -0,0 +1,68 @@
+---
+title: About the aws_ssm_resource_compliance_summaries Resource
+platform: aws
+---
+
+# aws\_ssm\_resource\_compliance\_summaries
+
+Use the `aws_ssm_resource_compliance_summaries` InSpec audit resource to test properties of a collection of AWS SSM compliance summaries.
+
+## Syntax
+
+ Ensure you have exactly 3 SSM Resource Compliance Summary Types
+
+    describe aws_ssm_resource_compliance_summaries do
+      its('compliance_types.count') { should cmp 3 }
+    end
+    
+#### Parameters
+
+This resource does not expect any parameters.
+
+See also the [AWS documentation on SSM](https://docs.aws.amazon.com/systems-manager/?id=docs_gateway).
+
+## Properties
+
+|Property                     | Description|
+| ---                         | --- |
+|compliance_types             | Provides the compliance type. |
+|compliant_summaries          | Provides a list of items that are compliant for the resource. |
+|execution_summaries          | Provides information about the execution |
+|non_compliant_summaries      | Provides a list of items that aren't compliant for the resource. |
+|overall_severity             | Provides the highest severity item found for the resource. |
+|resource_ids                 | Provides the resource id. |
+|resource_types               | Provides the resource type. |
+|status                       | Provides the compliance status for the resource. |
+
+For a comprehensive list of properties available, see [the API reference documentation](https://docs.aws.amazon.com/systems-manager/latest/APIReference/API_ResourceComplianceSummaryItem.html)
+
+## Examples
+
+##### Ensure Compliance Type of a SSM Resource Compliance Summary exists
+    describe aws_ssm_resource_compliance_summaries do
+      its('compliance_type') { should include 'ssm-compliance-type' }
```

Comment:
Typo: `compliance_type` => `compliance_types`.

Reply:
Updated in recent commit.